script/fetch-issues.js

```diff
@@ -141,14 +148,72 @@ async function fetchNextPage(owner, name, isIssue, data, cursor, pageCount) {
         }
       };
       if (!isIssue) {
+        let raw_comments, comments;
+        // collect review comments
+        raw_comments = node.reviews.nodes.map(node => node.comments.nodes);
+        raw_comments = [].concat.apply([], raw_comments);
+        // collect issue comments
+        raw_comments = raw_comments.concat(node.comments.nodes);
+        // filter useless reviews
+        raw_comments = raw_comments.filter(node => {
+          let flag = true;
+          // filter comments that don't need meta-reviews
+          if (node.bodyText.search('@gitmate-bot') !== -1) {
+            // gitmate-bot instructions
+            flag = false;
+          } else if (node.bodyText.search('@rultor') !== -1) {
+            // rultor instructions
+            flag = false;
+          } else if (node.bodyText.match(/^(unack|ack)/g)) {
+            // ack and unack messages
+            flag = false;
+          } else if (node.author && node.author.login === 'gitmate-bot') {
+            // comments by gitmate-bot
+            flag = false;
+          } else if (node.author && node.author.login === 'rultor') {
```

Comment:
This is a rather inelegant, maybe put up an array of author and comment body to ignore?

We fill it with sensible default, and make it so we can add more via env var.